core/tests/protractor_desktop/publicationAndLibrary.js

```diff
@@ -171,6 +174,32 @@ describe('Library index page', function() {
     users.logout();
   });
 
+  it('should save exploration to play later list', function() {
+    var EXPLORATION_FRACTION = 'fraction';
+    var CATEGORY_ARCHITECTURE = 'Architecture';
+    var LANGUAGE_ENGLISH = 'English';
+    var EXPLORATION_OBJECTIVE = 'hold the light of two trees';
+    learnerDashboardPage = new LearnerDashboardPage.LearnerDashboardPage();
```

Comment:
Do this line in the beforeEach() section.

Reply:
Done!